src/rest-server/docs/swagger.yaml

```diff
@@ -1317,165 +1317,31 @@ paths:
                   $ref: '#/components/responses/NoJobConfigError/content/application~1json/examples/NoJobConfigError'
         500:
           $ref: '#/components/responses/UnknownError'
-  /api/v2/storage/server/{storage}:
+  /api/v2/storages:
     get:
       tags:
       - storage
-      summary: Get storage server data in the system.
-      description: Get storage server data in the system.
-      operationId: getStorageServer
+      summary: Get storage list (persistent volume claims) for current user.
+      description: Get storage list for which current user has permissions.
+      operationId: getStorages
       security:
         - bearerAuth: []
-      parameters:
-      - $ref: '#/components/parameters/storage'
-      responses:
-        200:
-          description: Succeeded
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Storage'
-        500:
-          $ref: '#/components/responses/UnknownError'
-    delete:
-      tags:
-      - storage
-      summary: Remove storage server in the system.
-      description: Remove storage server in the system. Storage server empty is system reserved and cannot be removed.
-      operationId: removeStorageServer
-      security:
-        - bearerAuth: []
-      parameters:
-      - $ref: '#/components/parameters/storage'
-      responses:
-        200:
-          description: Succeeded
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Response'
-              example:
-                message: Storage Server is deleted successfully
-        403:
-          description: ForbiddenUserError or ForbiddenKeyError
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Response'
-              examples:
-                ForbiddenUserError:
-                  $ref: '#/components/responses/ForbiddenUserError/content/application~1json/examples/ForbiddenUserError'
-                ForbiddenKeyError:
-                  $ref: '#/components/responses/ForbiddenKeyError/content/application~1json/examples/ForbiddenKeyError'
-        500:
-          $ref: '#/components/responses/UnknownError'
-  /api/v2/storage/server:
-    get:
-      tags:
-      - storage
-      summary: Get storage server data in the system.
-      description: Given storage server names, find server data.
-      operationId: getStorageServers
-      security:
-        - bearerAuth: []
-      parameters:
-      - name: names
-        in: query
-        description: filter storage server with names, default name empty will be ignored
-        schema:
-          type: string
       responses:
         200:
           description: Succeeded
           content:
             application/json:
               schema:
-                type: array
```

Comment:
It's recommended to return object always, because array cannot be extended if you want to add more information in the future, here's [a discussion](https://softwareengineering.stackexchange.com/a/304638)